uktrade/data-hub-frontend
# Rebuild OMIS subscribers form in React (remove vue.js)

## Description of change

As part of the work to remove the Vue.js Typeahead, we need to refactor the OMIS add subscribers form.

## Test instructions

You should be able to add/edit subscribers (AKA 'Advisers in the UK') for an OMIS order.

## Screenshots
### Before

<img width="746" alt="Screenshot 2022-02-11 at 15 58 52" src="https://user-images.githubusercontent.com/36161814/153625078-cbb16c02-d796-4ae6-af4b-62978e443695.png">

### After

<img width="992" alt="Screenshot 2022-02-11 at 15 58 46" src="https://user-images.githubusercontent.com/36161814/153625064-92eb1e73-3f06-4414-a058-f38a56e67404.png">

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
